### PR TITLE
jbao-huntsville

### PR DESCRIPTION
**Pull request by baojason**
jbao-huntsville

### CODE REVIEW

**Comment by jaap15** [comment]
Nice job.

File: ccoole-Tuscaloosa,Alabama.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Tuscaloosa, Alabama
```
Comment (jaap15) [inline ccoole-Tuscaloosa,Alabama.md:1]: Roll Tide?

**Comment by hectorsector** [comment]
@baojason you opened this PR in reverse, you can close it, and start a new one. Make sure `base: master` and `compare: jbao-huntsville`. Open up a new issue if you need help!

**Review by jaap15 — APPROVED** [review]
this is great work . go ahead and :ship:

**Pull request merged**